test/__init__.py

```diff
@@ -228,28 +228,3 @@ def __init__(self):
 
     def emit(self, record):
         self.records.append(record)
-
-
-class LogRecorder(object):
-    def __init__(self, target=logging.root):
-        super(LogRecorder, self).__init__()
-        self._target = target
-        self._handler = _ListHandler()
```

Comment:
Please remove `_ListHandler` too (the class just above)

Reply:
done